wca-guidelines.md

```diff
@@ -180,6 +180,7 @@ To be more informative, each Guideline is classified using one of the following
 - B1+) [REMINDER] The competitor must use a puzzle without textures, markings, or other features that distinguish similar pieces (see [Regulation 3k](regulations:regulation:3k)). This should be given special attention for Blindfolded Solving.
 - B1b+) [RECOMMENDATION] Blindfolds should be checked by the WCA Delegate before use in the competition.
 - B2d+) [ADDITION] By default, the competitor starts the solve the first time that they start the timer after the judge has indicated that they are ready (see [Regulation B2a](regulations:regulation:B2a)). If they want to check that the timer is in working order, they must (ask and) receive confirmation from the judge each time before starting/resetting the timer during this phase. Penalty for starting and resetting a timer without confirmation from the judge: disqualification of the attempt (DNF).
+- B4c+) [ADDITION] An attempt should not be disqualified solely due to the judge not putting a sight blocker. However, the WCA Delegate may replace the attempt with an extra attempt if they have a reason to suspect that the competitor gained an unfair advantage.
```

Comment:
I think this needs to make explicit that this does *not* apply if https://www.worldcubeassociation.org/regulations/#B4c3 is in effect.

Reply:
> thus making it look to have the same level of importance.

What would be the problem with that?

Reply:
> What would be the problem with that?

A competition with all scramblers signatures missing won't probably cause a debate on whether the results should stay or not, but I don't think it would be fine to have the same for the sight blockers.

Reply:
Understood. What about changing "should not be disqualified" to "May remain valid" (or something like that). With that wording, it would be a possibility instead of a recommendation.

Reply:
It would sound a way better. But maybe it should even be reworded to make it an exception? To clearly define that it's not a normal case and should be trated seriously.